packages/sit-onyx/src/components/OnyxStepper/OnyxStepper.ct.tsx

```diff
@@ -555,3 +555,21 @@ test("should format zero with precision", async ({ mount }) => {
   await expect(input).toHaveValue("0.00");
   await expect(() => expect(modelValue).toBe(0)).toPass();
 });
+
+test("should format with formatNumber", async ({ page, mount }) => {
+  // ARRANGE
+  const component = await mount(
+    <OnyxStepper
+      label="Label"
+      modelValue={1000000}
+      formatNumber={true}
+      style={{ width: "16rem" }}
+    />,
+  );
+  const input = component.getByLabel("Label");
+  const display = component.locator(".onyx-stepper__display");
+  // // ASSERT
+  await expect(input).toHaveValue("1000000");
+  await expect(display).toHaveText("1,000,000");
+  await expect(page).toHaveScreenshot("formatNumber.png");
```

Comment:
The screenshots contain a lot of whitespace, we can reduce the size by only considering the component itself for the screenshot.
```suggestion
  await expect(component).toHaveScreenshot("formatNumber.png");
```